easybuild/framework/easyconfig/format/yeb.py

```diff
@@ -127,3 +127,13 @@ def is_yeb_format(filename, rawcontent):
             if line.startswith('name: '):
                 isyeb = True
     return isyeb
+
+# join functionality:
```

Comment:
description of the function should indeed go into the docstring, references to details w.r.t. the implementation should be comments